boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuflpr_0_7_0.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54l15pdk_nrf54l15_common_0_3_0.dtsi"
```

Comment:
Same with 0.3.0 vs 0.7.0

Reply:
Eventually to name it for example config_v1 like: nrf54l15pdk_nrf54l15_common_config_v1.dtsi and include it into 0.3.0 and 0.7.0